frontend/micro-ui/web/micro-ui-internals/packages/libraries/src/hooks/useApplicationForBillSearch.js

```diff
@@ -116,7 +116,7 @@ export const useApplicationsForBusinessServiceSearch = ({ tenantId, businessServ
   if (window.location.href.includes("chb-services")) {
     _key = "chb"
   } 
-  if (window.location.href.includes("ads-services")) {
+  if (window.location.href.includes("adv-services")) {
```

Comment:
Why we have added this line add description

Reply:
it will check if the url contains adv-services then it will call the particular object of adv, which then call the search function for the bill

i have not added comments here as it is a common file